notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2216-Slew_and_Track.ipynb

```diff
@@ -0,0 +1,827 @@
+    "- Get M2 hex (simulator) Ready: check config; make sure LUT is on and has valid inputs; make sure hex is at LUT position  \n",
+    "- Finally, get the MTAOS CSC ready  \n",
+    "\n",
+    "[LVV-T2344]: https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T2344"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b4ec348",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logging.basicConfig(format=\"%(asctime)s %(name)s: %(message)s\", level=logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5cbe1cee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "log = logging.getLogger(\"setup\")\n",
+    "log.level = logging.DEBUG"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "37ef054e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"LSST_DDS_HISTORYSYNC\"] = \"200\"\n",
+    "domain = salobj.Domain()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3ffca8f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mtcs = MTCS(domain=domain, log=log)\n",
+    "mtcs.set_rem_loglevel(logging.ERROR)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bed75413",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.start_task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fed1ffd-9a1c-4ad0-bf60-b2fe21218e87",
+   "metadata": {},
+   "source": [
+    "The cell below exposes the mount so we have a more compact code when moving the telescope carefully while M1M3 and M2 are using it to calculate their look-up tables instead of using their internal inclinometer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0af52d5f-623a-4ee1-87eb-d9d86c7889f7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mtmount = mtcs.rem.mtmount"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d5cbd3a5-409d-4050-8e2a-7789d9bbd46a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script = salobj.Controller(\"Script\", index=42658885)\n",
+    "await asyncio.sleep(10) \n",
+    "\n",
+    "script.start_task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07a1c841-3769-4c01-9c04-ee2c10f037a5",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "Add a filter to all the loggers to hide the DDS read queue is filling / full."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2d330bc0-b545-4e3f-9bc8-63bee92c3af7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vandv.logger.add_filter_to_mtcs()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78dc7c29-4e21-4f10-952f-0d238f48987d",
+   "metadata": {},
+   "source": [
+    "## Slewing without LUT corrections\n",
```

Comment:
Is there any command/event or in the EUI to ensure no LUT is applied?

Reply:
For M1M3 and M2, the LUT is always applied. However, on Level 3, we say that there are not being applied when we have the LUT using the elevation values from the inclinometer since they do not change. And we say that the LUT is active when it uses the mount's elevation values. That does not make much sense when running on TTS, but it makes the difference on Level 3.